plugins/askrene/askrene.c

```diff
@@ -399,48 +382,17 @@ static const char *get_routes(const tal_t *ctx,
 
 		/* Clear any entries in capacities array if we
 		 * override them (incl local channels) */
-		layer_clear_overridden_capacities(l, askrene->gossmap, rq->capacities);
```

Comment:
This patch does contain a number of reformats which add noise, making it harder to review.

Reply:
OK. I have reverted back the reformats.